core/src/main/java/com/datastrato/gravitino/storage/relational/mysql/MySQLBackend.java

```diff
@@ -25,41 +38,207 @@ public class MySQLBackend implements RelationalBackend {
 
   /** Initialize the MySQL backend instance. */
   @Override
-  public void initialize(Config config) {}
+  public void initialize(Config config) {
+    SqlSessionFactoryHelper.getInstance().init(config);
+  }
 
   @Override
   public <E extends Entity & HasIdentifier> List<E> list(
-      Namespace namespace, Entity.EntityType entityType) throws NoSuchEntityException {
-    throw new UnsupportedOperationException("Unsupported operation now.");
+      Namespace namespace, Entity.EntityType entityType) {
+    try (SqlSession session = SqlSessions.getSqlSession()) {
```

Comment:
Can we remove the session since it hasn't been used?

Reply:
Using the try-with-resources statement, it can be automatically closed after use is completed. This is to provide a guarantee.